setup.py

```diff
@@ -1,14 +1,19 @@
 """Python packaging for PDBx/mmCIF utilities."""
 import setuptools
-import versioneer
+
+# NOTE: The following reads the version number and makes
+#       if available to the packaging tools before installation.
+#       REF: https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package  # noqa: E501
+exec(open("pdbx/version.py").read())
```

Comment:
should this use a file handle context manager?
```python
with open("pdbx/version.py") as fh:
    exec(fh.read())
```

Reply:
Yes, thank you.